### PR TITLE
Explain how to deploy to ZEIT Now

### PR DESCRIPTION
**Pull request by leo**
Now that ZEIT has launched [zero config deployments](https://twitter.com/zeithq/status/1159145442896166912), deploying a Ember site is now much easier:

image

### CODE REVIEW

**Comment by leo** [comment]
In the CI, the following error is showing - can someone help me understand what the problem is?

image

**Comment by MelSumner** [comment]
If we decide to add this, we should also add sections- or at least issues - on what other deployment options are available. @ember-learn/learning-core-team

**Comment by leo** [comment]
@MelSumner 👋 Do you have any idea why my test is failing? I only added more markdown.

**Comment by leo** [comment]
@MelSumner Any chance we could get this merged? 😊 👋

**Comment by rwwagner90** [comment]
@leo as Mel said, if we decided to merge this, we would also need to add sections on other deployment options, rather than endorsing just one. We can try to discuss it at this week's learning team meeting.

**Comment by leo** [comment]
> We can try to discuss it at this week's learning team meeting.

Got it. Thank you!

File: guides/release/tutorial/deploying.md [hunk 1]
```diff
@@ -32,6 +32,34 @@ You can deploy your application to any web server by copying the output from `em
 scp -r dist/* myserver.com:/var/www/public/
 ```
```
Comment (MelSumner) [inline guides/release/tutorial/deploying.md]: Please insert : 

```
## Deployment Options

There are many different deployment options for your Ember application. The following are a few options as submitted by community members. 
```

File: guides/release/tutorial/deploying.md [hunk 2]
```diff
@@ -32,6 +32,34 @@ You can deploy your application to any web server by copying the output from `em
...
 
+## Deploying to ZEIT Now
```
Comment (MelSumner) [inline guides/release/tutorial/deploying.md]: - Change the level 2 heading to a level 3 heading
- change the text to ZEIT Now
- place in alphabetical order with the other options

File: guides/release/tutorial/deploying.md [hunk 3]
```diff
@@ -32,6 +32,34 @@ You can deploy your application to any web server by copying the output from `em
...
+now
+```
+
+**Alternatively**, you can also use their integration for [GitHub](https://zeit.co/github) or [GitLab](https://zeit.co/gitlab).
+
+That’s all!
+
+Your site will deploy, and you will receive a link similar to the following: https://ember.now-examples.now.sh
+
 ## Deploying to surge.sh
```
Comment (MelSumner) [inline guides/release/tutorial/deploying.md]: - Change from a level 2 heading to a level 3 heading
- change text to "Surge"

**Comment by MelSumner** [comment]
Ok @leo @rwwagner90  I looked into this more and I think I figured out how we can accept this PR. @leo if you can please make the requested changes on the PR and I'll merge it.

**Comment by chancancode** [comment]
FWIW, I don’t think we necessarily have to sign *ourselves* up to do the work to include other vendors. We just have to decide what the general policy is in terms of documenting third party things (eg whether we are open to them at all, whether they have a free tier without requiring a credit card, should we just ask them to host and maintain the documentation themselves and just link to them). Vendors or users who want to champion for those vendors can then just submit patches themselves following those guidelines. If we only end up documenting a single vendor because no one else did the work, I don’t think that’s necessarily a problem on our end.

**Comment by MelSumner** [comment]
@chancancode agree- once I re-read the page content I saw other options, which is why I requested a few changes before merging 👍

File: guides/release/tutorial/deploying.md (hunk 1 above)
Comment (leo) [reply guides/release/tutorial/deploying.md]: Done

File: guides/release/tutorial/deploying.md (hunk 2 above)
Comment (leo) [reply guides/release/tutorial/deploying.md]: Done

File: guides/release/tutorial/deploying.md (hunk 3 above)
Comment (leo) [reply guides/release/tutorial/deploying.md]: Done

**Comment by leo** [comment]
Thanks for the requested changes, @MelSumner! I've applied them!

**Comment by leo** [comment]
No problem, @MelSumner! Anything else missing before it can be merged?

**Comment by MelSumner** [comment]
This is failing because of a dependency that needs updating, I think, but I just tried updating the dependencies locally and couldn't find one that needed updating. I think I need to call in @locks @jenweber @rwwagner90 on this.

**Comment by rwwagner90** [comment]
@MelSumner I will take a look 👍

**Comment by rwwagner90** [comment]
@MelSumner How do I pull this down locally? The PR says it is from `unknown repository`, so not sure where the branch is to check out.

**Comment by MelSumner** [comment]
@leo Can you connect with @rwwagner90 so we can figure out what dependency needs to be updated? Thanks!

**Comment by locks** [comment]
@rwwagner90 has reviewed and merged my PR that updates the dependency, if this branch is rebased we should be good to go!

**Comment by leo** [comment]
@rwwagner90 You should be able to click "Update branch" on the GitHub UI below? I unfortunately accidentally deleted my fork, so only the PR exists now.

But I think we can update it through the GitHub UI:

**Comment by rwwagner90** [comment]
@leo I see no update button, unfortunately.

**Comment by leo** [comment]
@rwwagner90 I created it again as https://github.com/ember-learn/guides-source/pull/1043!

**Pull request closed without merging**